.changeset/spotty-buckets-hang.md

```diff
@@ -0,0 +1,33 @@
+---
+"@biomejs/biome": major
+---
+
+Reworked some recommended rules recommended to be less pedantic and blocking. This is a **breaking change** if your project relied on those rules to block the CI in case of violations, you should raise their diagnostic to **error**.
+
+Some rules aren't recommended anymore, and some others return a different severity:
+
+The following rules return a **warning** diagnostic:
+- `noDelete`
+- `noForEach`
+- `noSuspiciousSemicolonInJsx`
+- `noThisInStatic`
+- `noUnusedLabels`
+
+The following rules return an **information** diagnostic:
+- `noUselessCatch`
+- `noUselessConstructor`
+- `noUselessEmptyExport`
+- `noUselessFragments`
+- `noUselessLabel`
+- `noUselessLoneBlockStatements`
+- `noUselessSwitchCase`
+- `noUselessTernary`
+- `noUselessThisAlias`
+- `noUselessTypeConstraint`
+
+The following rules aren't recommended anymore:
+- `noDelete`
+- `noForEach`
+- `noFlatMapIdentity`
```

Comment:
Why this rule is no longer recommended?

Reply:
clippy has this rule under the pedantic category, and it doesn't recommended it, I think we should follow their lead and stick with it. 

Our recommendation set is very opinionated and we should reduce it

Reply:
Many users like the fact that Biome is opinionated.
I accept that we could be more cautious about what we add in the recommended rule set.
Personally I could keep this rule in the recommended set because it brings one of the mission of Biome: making dev better dev by helping to learn JS API.
We could reduce the severity level to `information`.

Reply:
Sounds good!